lang/en.yml

```diff
@@ -72,6 +72,7 @@ en:
     ParentMode_child: 'Under another page'
     ParentMode_top: 'Top level'
     Title: 'Add page'
+    ROOT_BREADCRUMB_TITLE: 'Pages'
```

Comment:
Duplicate of `SilverStripe\CMS\Controllers\CMSPagesController.MENUTITLE`. Please use the existing key instead.

Reply:
Update made